core/src/main/java/io/questdb/griffin/SqlCompilerImpl.java

```diff
@@ -2812,11 +2824,12 @@ private void executeCreateMatView(CreateMatViewOperation createMatViewOp, SqlExe
                     CharSequence volumePath = configuration.getVolumeDefinitions().resolveAlias(volumeAlias);
                     if (volumePath != null) {
                         if (!ff.isDirOrSoftLinkDir(path.of(volumePath).$())) {
-                            throw CairoException.critical(0).put("not a valid path for volume [alias=")
+                            throw CairoException.critical(0).position(createMatViewOp.getVolumePosition())
```

Comment:
Why does this have to be a critical `CairoException`?

Reply:
This is the same for CREATE TABLE (line 2929 in this class).
I can see why this is CairoException.
It is a config issue, and not SQL syntax.
Not sure about having it critical, but probably that is ok too.
The misconfiguration should be fixed ASAP, so critical probably makes sense.